### PR TITLE
 configuration file added, architecture updated according to config file,

### PR DESCRIPTION
**Pull request by hopetes**
Configuration file and, Config Class is added.

### CODE REVIEW

File: pulse-physiology_FHIR/Diagrams/usecase_diagram.png
Comment (basbergevoet) [inline pulse-physiology_FHIR/Diagrams/usecase_diagram.png:1]: @hopetes I am not going to block the review with this remark, but still I want to share it with you as a hint and a potential improvement: Now that you have the class diagram as well, wouldn't you want to reconsider/simplify the package diagram? Currently you show components in the packages, that are very much resembling the classes in the class diagram. Do you really need to model these components? Why not just three packages, and in the class diagram zoom in to the relevant package(s) and show the class diagram structure that is inside those packages? Just for keeping things simple and more readable for the user. 

I still would advice you to use more descriptive package names. "Pulse Engine Interface to FHIR Software" is not covering everything that the package does. Yes, it interfaces to FHIR. But it also interface and controls the PPE engine. And the package name "External systems" is probably the most generic name I could think of..... It is always correct, no matter what is actually in the package :-). For self-explaining designs, naming of the objects in those designs is really making a difference, if you do it right!

File: pulse-physiology_FHIR/Diagrams/class_diagram.png
Comment (basbergevoet) [inline pulse-physiology_FHIR/Diagrams/class_diagram.png:1]: @hopetes Now this package name I really like! It is super descriptive and to the point!

File: pulse-physiology_FHIR/config.ini
Comment (basbergevoet) [inline pulse-physiology_FHIR/config.ini:1]: @hopetes Please update the file as you did in our standup today (July 15th), so taking out the database properties as you are not using them.

**Review by basbergevoet — APPROVED** [review]

File: pulse-physiology_FHIR/Diagrams/usecase_diagram.png
Comment (hopetes) [reply pulse-physiology_FHIR/Diagrams/usecase_diagram.png:1]: Noted!

File: pulse-physiology_FHIR/config.ini
Comment (basbergevoet) [reply pulse-physiology_FHIR/config.ini:1]: @hopetes You say you will update it, but the file is still unchanged.....

**Pull request merged**